### PR TITLE
Feature/variant grid home

### PR DESCRIPTION
**Pull request by abdullacontractor**
Create basic variant grid on homepage
localhost_9000__channel_id=5_

### CODE REVIEW

File: frontend/app/routes/home.js [hunk 1]
```diff
@@ -1,7 +1,29 @@
...
+    }
+  };
+
+  async model(params) {
+    let channelReferences = await this.store.query('channel-reference', {
+      channel_id: params.channel_id,
+      owner_type: 'Variant',
+      limit: 250,
+    });
+    let variantIds = channelReferences.filterBy('ownerType', 'Variant').mapBy('ownerId');
```
Comment (billybonks) [inline frontend/app/routes/home.js:17]: can remove the filterby

Comment (abdullacontractor) [reply frontend/app/routes/home.js:17]: right, yes we can

File: frontend/app/controllers/home.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  @service store;
+  queryParams = ['channel_id']
+
+  async fetchChannel() {
+    if (this.channel_id) {
+      let channels = await this.store.query('channel', {
+        ids: [this.channel_id],
+      });
+      let channel = channels.firstObject;
+      this.setProperties({ channel });
```
Comment (billybonks) [inline frontend/app/controllers/home.js:14]: this.channel = channel

File: frontend/app/controllers/home.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import Controller from '@ember/controller';
+import { inject as service } from '@ember/service';
+
+export default class OrdersPickController extends Controller {
```
Comment (billybonks) [inline frontend/app/controllers/home.js:4]: OrdersPickController

**Review by billybonks — APPROVED** [review]
👯

File: frontend/app/controllers/home.js (hunk 2 above)
Comment (abdullacontractor) [reply frontend/app/controllers/home.js]: That actually does not work, the frontend didnt respond to the channel change. `this.set('channel', channels.firstObject);` works as expected though.

**Pull request merged**